exercises/sum.cpp

```diff
@@ -6,3 +6,19 @@
   Insert the second number: 2
   Sum: 3
 */
+
+#include <iostream>
+
+int main(){
+  int a, b;
+  
+  std::cout << "Insert the first number: ";
+  std::cin >> a;
+
+  std::cout << "Insert the second number: ";
+  std::cin >> b;
+
+  std::cout << "Sum: " << a + b << std::endl;
```

Comment:
potresti usare `using namespace std;` per evitare di scrivere sempre `std::`.

inoltre, potresti evitare un po' di ridondanza nel codice sopra estraendo l'inserimento del numero in una funzione, vedi questa PR https://github.com/Helias/git-course-exercises/pull/12/files